code/modules/cargo/exports/organs.dm

```diff
@@ -36,3 +36,15 @@
 	cost = 5
 	unit_name = "humanoid tounge"
 	export_types = list(/obj/item/organ/tongue)
+
+/datum/export/organ/tail/lizard
+	cost = 250
+	unit_name = "lizard tail"
+	export_types = list(/obj/item/organ/tail/lizard)
+
+
+/datum/export/organ/tail/cat
+	cost = 500
```

Comment:
suggestion: reduce this to 250 (or maybe 300, since I feel like cat tails would be more valuable than lizard tails), but let cargo sell cat EARS for 250 as well